src/accounts.cs

```diff
@@ -112,6 +113,12 @@ interface ACAccountStore {
 		[Wrap ("RequestAccess (accountType, options == null ? null : options.Dictionary, completion)")]
 		[Async]
 		void RequestAccess (ACAccountType accountType, [NullAllowed] AccountStoreOptions options, ACRequestCompletionHandler completion);
+
+		// For some reason this was never added before, xtro caught it.
```

Comment:
Nope, it was inside unclassified 

`common.unclassified:!missing-selector! ACAccountStore::removeAccount:withCompletionHandler: not bound`